Looks like some of the OSX tests are failing:

```
../.libs/libnfft3_threads.dylib -L/Users/travis/miniconda3/envs/_build/lib /Users/travis/miniconda3/envs/_build/lib/libfftw3_threads.dylib /Users/travis/miniconda3/envs/_build/lib/libfftw3.dylib -lpthread -lm /Users/travis/miniconda3/envs/_build/lib/libcunit.dylib -L/Users/travis/miniconda3/conda-bld/work/CUnit-2.1-3/CUnit/Sources -lc -lncurses -fopenmp
ld: warning: directory not found for option '-L/Users/travis/miniconda3/conda-bld/work/CUnit-2.1-3/CUnit/Sources'
/Applications/Xcode.app/Contents/Developer/usr/bin/make  check-TESTS
../config/test-driver: line 95: 22961 Trace/BPT trap: 5       "$@" > $log_file 2>&1
FAIL: checkall
../config/test-driver: line 95: 22981 Trace/BPT trap: 5       "$@" > $log_file 2>&1
FAIL: checkall_threads
Making all in data
make[6]: Nothing to be done for `all'.
make[6]: Nothing to be done for `all-am'.
============================================================================
Testsuite summary for NFFT 3.3.1
============================================================================
# TOTAL: 2
# PASS:  0
# SKIP:  0
# XFAIL: 0
# FAIL:  2
# XPASS: 0
# ERROR: 0
============================================================================
See tests/test-suite.log
Please report to mail@nfft.org
============================================================================
make[4]: *** [test-suite.log] Error 1
make[3]: *** [check-TESTS] Error 2
make[2]: *** [check-am] Error 2
make[1]: *** [check-recursive] Error 1
make: *** [check-recursive] Error 1
Command failed: /bin/bash -x -e /Users/travis/build/conda-forge/nfft-feedstock/recipe/build.sh

```

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -2,6 +2,11 @@
 
 ./bootstrap.sh
 
+if [[ `uname` == 'Darwin' ]]; then
+    # make check below fails on osx unless $PREFIX/lib is added to rpath
+    LDFLAGS="$LDFLAGS -Wl,-rpath,${PREFIX}/lib"
+fi
+
 ./configure --prefix=$PREFIX \
             --enable-applications \
             --enable-all \

